src/uct/ib/ud/base/ud_iface.h

```diff
@@ -113,6 +113,9 @@ struct uct_ud_ctl_hdr {
         } conn_rep;
         uint32_t data;
     };
+#if ENABLE_DEBUG_DATA == 1
```

Comment:
we need to keep same wire protocol between debug and release versions (consider client/server application)